embedded-graphics/src/style/mod.rs

```diff
@@ -6,4 +6,106 @@ mod text_style;
 
 pub use primitive_style::{PrimitiveStyle, PrimitiveStyleBuilder};
 pub use styled::Styled;
-pub use text_style::TextStyle;
+pub use text_style::{TextStyle, TextStyleBuilder};
+
+/// Create a [`PrimitiveStyle`]
+///
+/// Any properties not specified in the macro call will use the values provided by
```

Comment:
This should also mention what properties can be specified like in the `text_style` macro below.